sm/sm.mk.in

```diff
@@ -18,10 +18,11 @@ sm_c_srcs = \
   ed25519/fe.c \
   ed25519/ge.c \
   ed25519/sc.c \
+  soc/@TARGET_PLATFORM@/@TARGET_PLATFORM@.c \
```

Comment:
Can we do more general name (e.g., platform) maybe?

Reply:
I mean instead of soc

Reply:
sure, I didn't like the options I'd thought of other than soc but you are probably right that platform is better. I'll switch all usages to that